File: 算法基础课(修订版)/笔试拿分技巧/P14163/verify_acm.py
class Solution:
    def solve(self, a):
        return " ".join(str(x) for x in a)


if __name__ == "__main__":
    n = int(input())
    a = list(map(int, input().split()))

    solution = Solution()
    print(solution.solve(a), end="")

File: 算法基础课(修订版)/笔试拿分技巧/P14163/verify_acm.cc
#include <iostream>
#include <sstream>
#include <vector>
using namespace std;

class Solution {
public:
    string solve(vector<int>& a) {
        ostringstream oss;
        for (int i = 0; i < (int)a.size(); ++i) {
            if (i > 0) {
                oss << ' ';
            }
            oss << a[i];
        }
        return oss.str();
    }
};

int main() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; ++i) {
        cin >> a[i];
    }

    Solution solution;
    cout << solution.solve(a);
    return 0;
}
